src/constants.js

```diff
@@ -50,4 +50,20 @@ const MIME_TYPES = {
   md: "text/markdown",
 };
 
-module.exports = { WS_EVENTS, MIME_TYPES };
+const TEMPLATES = [
```

Comment:
How about making this variable as a global constant ?  https://github.com/ameerthehacker/blazepack/blob/master/bin/blazepack.js#L26

Otherwise we have to manage 2 variables whenever we need to update templates.

Reply:
Right!

Reply:
Should i move the `TEMPLATES` from blazepack.js to constants.js?

Reply:
https://github.com/ameerthehacker/blazepack/blob/master/bin/blazepack.js#L42
reuse the TEMPLATES in similar places, I would suggest it to remain inside constant.js itself global variables can become ghosts

Reply:
Ok, let me move it.